crt_portal/cts_forms/views.py

```diff
@@ -70,11 +70,7 @@ def error_403(request, exception=None):
 def error_404(request, exception=None):
     return render(
         request,
-        'forms/errors.html', {
-            'status': 404,
-            'message': _("We can't find the page you are looking for"),
-            'helptext': _("Try returning to the previous page")
-        },
+        'forms/error_404.html',
```

Comment:
Wondering why we chose to make a new error page instead of re-using what we have (`forms/errors.html`)?

It looks like we have spanish translations (and the wording is the same) so we could wrap these sentences into `trans` blocks.

Reply:
@cantsin I thought about this as well. One of reasons I decided on a separate file is in the acceptance criteria nice to have some of those items would be better implemented in a separate page. Plus looking at cards 706 and 707 on ZenHub, these cards have different design mocks. However, Looking at them closer I can see that we can still just update our current errors.html template and use it for this card and 706 and 707. I'd love to hear your feedback on if this is the better implementation going forward.

Reply:
I like the idea of updating our `errors.html` template because that will improve all errors in one fell swoop, and not just the 404 error. If we could do that, that'd be great! (Also I think doing so may make 706/707 work easier?)

An alternative is to add a custom block in `errors.html` that we can override, if the template doesn't work for us for whatever reason. Then we can do `{% extends error.html %}` in `error_404.html` and add the new block.

Reply:
Thanks for the suggestion. I spoke with Randy and Bill about what the general look and feel should be and they suggested that card 706, and 707 mocks is what the goal is. So with that in mind I updated the styling as well as moving the 404 back to using `errors.html`.